frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -325,133 +404,183 @@ export const OperandList: React.FC<OperandListProps> = (props) => {
       data={data}
       EmptyMsg={EmptyMsg}
       aria-label="Operands"
-      Header={Header}
-      Row={OperandTableRow}
+      Header={managesOtherNamespaces && props.toggleView ? AllNsHeader : CurrentNsHeader}
```

Comment:
nit, already destructured
```suggestion
      Header={managesOtherNamespaces && toggleView ? AllNsHeader : CurrentNsHeader}
```